backend/migrations/README.md

```diff
@@ -0,0 +1,84 @@
+# Managing DB with Alembic locally
+
+When we run ```docker-compose up``` the postgres container is created with no tables or schemas.
```

Comment:
Maybe we should change the command here to `docker-compose build` because `docker-compose up` builds and starts the containers, so after running it the tables will be created with `declarative_base`